daemon/containerd/image_test.go

```diff
@@ -0,0 +1,298 @@
+package containerd
+
+import (
+	"context"
+	"io"
+	"math/rand"
+	"path/filepath"
+	"testing"
+
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/metadata"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/log/logtest"
+	"github.com/distribution/reference"
+	dockerimages "github.com/docker/docker/daemon/images"
```

Comment:
Let's keep the aliases consistent:
```suggestion
	dimages "github.com/docker/docker/daemon/images"
```